dotcom-rendering/src/lib/assets.ts

```diff
@@ -38,15 +38,24 @@ export const loadableManifestJson = loadableManifest;
 export const getScriptArrayFromFilename = (
 	filename: string,
 ): { src: string; legacy: boolean }[] => {
+	// Get legacy file name if one's available
```

Comment:
Maybe worth mentioning the `INCLUDE_LEGACY` here so people know where to look for this when coming back to this?